pkgs/by-name/hypercore/dream2.nix

```
@@ -6,7 +6,7 @@
 }:
 rec {
   name = "hypercore-${version}";
-  version = "10.28.11";
+  version = "11.1.1";
```

Another release `11.1.2` is out which fixes the tests. Please change the version and regenerate the `src` hash and the lockfile.

```suggestion
  version = "11.1.2";
```